model/model.js

```diff
@@ -78,4 +70,22 @@ exports.selectArticleById = async (articleId) => {
     }
     return article.rows[0]
   })
+}
+
+exports.selectCommentsByArticleId = (articleId) => {
+  const articleIdNum = Number(articleId)
+  const regex = new RegExp(/[^\d]/g)
+  if (regex.test(articleIdNum)) {
```

Comment:
This is a good method, but it isn't necessary - PSQL should be able to handle this for you as it will throw an error if you try and put a non-number in there (you can keep it like this, but I strongly recommend that you have a [look at PSQL errors](https://notes.northcoders.com/courses/js-back-end/error-handling#psql-thrown-errors) and learn how they work so that this knowledge comes across to anyone who looks at your project